src/lib/Components/Buttons/DarkNavigationButton.tsx

```diff
@@ -13,37 +10,20 @@ interface Props extends React.Props<DarkNavigationButton> {
   style?: any
 }
 
-const BackgroundView = styled.View`
-  background-color: ${Colors.Black};
-  padding-left: 20px;
-  padding-top: 13px;
-  padding-bottom: 9px;
-  padding-right: 6px;
-`
-
-const Text = styled.Text`
-  font-family: "${Fonts.GaramondRegular}";
-  font-size: 15;
-  color: ${Colors.White};
-`
-
-const Row = styled.View`
-  flex-direction: row;
-  justify-content: space-between;
-`
-
 export default class DarkNavigationButton extends React.Component<Props, any> {
   render() {
     const showNavArrow = this.props.href || this.props.onPress
     return (
-      <BackgroundView style={this.props.style}>
```

Comment:
Seems like we're not using the `style` prop anymore. Will that affect anywhere else that's using this component? Should we remove it from the `Props` definition above?